app/index.html

```diff
@@ -161,11 +161,12 @@ <h3>Skills</h3>
                 <h2 id="form-text">I&apos;m always interested in hearing about new projects, so if you&apos;d like to chat please get in touch.</h2>
                 <form id="mobile-form" action="https://formspree.io/f/mvolzpve" method="post">
                     <input class="text-fields" type="text" id="fullname" name="full_name" placeholder="Full name" required maxlength="30">
-                    <input class="text-fields" type="email" id="email" name="email" placeholder="Email address" required>
+                    <input class="text-fields" id="email" name="email" placeholder="Email address">
                     <textarea class="text-fields primary-font" name="user_message" id="message" placeholder="Enter text here..." required maxlength="500"></textarea>
                     <div>
-                        <button class="button work-item-button primary-font" type="submit">Get in touch</button>
+                        <button id="form-btn" class="button work-item-button primary-font" type="submit">Get in touch</button>
                     </div>
+                    <small>Hello</small>
```

Comment:
# HOW AMAZING DESIGN 👏😃

It is not requirement. I just wish you luck 🍀. KEEP ON coding such an awesome projects

Reply:
Thanks a lot @Rustamxon7